src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -214,8 +214,8 @@ private function get_block_queries( WP_Block_Parser_Block $block ) {
 			$this->block_types = $this->config_option->get();
 		}
 
-		if ( isset( $block->blockName, $block->innerHTML ) && array_key_exists( $block->blockName, $this->block_types ) ) {
-			return $this->block_types[ $block->blockName ];
+		if ( isset( $block->blockName, $this->block_types[ $block->blockName ]['xpath'] ) ) {
```

Comment:
We were checking for `$block->innerHTML` here but we are inside the private method `get_block_queries`. This is not appropriate to me and I will reuse this method for the keys later.

I moved the check where it's really needed (and covered with tests).